js/less.js

```
@@ -7,15 +7,15 @@
  *
- * This is example of how to create less like command for jQuery Terminal
+ * This is an example of how to create less like command for jQuery Terminal
  * the code is based on the one from leash shell and written as jQuery plugin
  *
  * Copyright (c) 2018-2025 Jakub Jankiewicz <https://jcubic.pl/me>
  * Released under the MIT license
  *
  */
 /* global define */
-(function(factory, undefined) {
+(function (factory, undefined) {
```

Biome (2.3.14)

[error] Do not shadow the global "undefined" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)